### DISCUSSION TITLE
[LangChainChunkingError]: Unsupported file type, please check your file is a supported type, or just create an issue.

### DISCUSSION THREAD

**Discussion by harboot** [lobehub/lobehub#4250] [General]
[LangChainChunkingError]: Unsupported file type, please check your file is a supported type, or just create an issue.

**Comment by harboot** [comment]
![image](https://github.com/user-attachments/assets/9e0d9ac5-d2f2-4dc4-acb7-6f4043f8476c)

**Comment by SpeedupMaster** [comment]
Excel is not currently supported. 
#3550

**Comment by harboot** [comment]
Thank you
